src/index.html

```diff
@@ -11,7 +11,200 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
     <script type="text/javascript" src="./main.js"></script>
+
+    <form
+      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+      method="POST"
+      onsubmit="onSubmit()"
+    >
+      <fieldset class="fs">
+        <legend>Personal information</legend>
+
+        <div class="field">
+          <label for="surname" >
```

Comment:
Не залишай пусті поля перед закривачаючим тегом
```suggestion
          <label for="surname">
```